src/components/Person/Person.jsx

```diff
@@ -1 +1,13 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ name, age, sex, isMarried, partnerName }) => {
```

Comment:
The task requires the `Person` component to accept a single `person` object as a prop instead of individual fields. You should destructure the `person` object to access `name`, `age`, `sex`, `isMarried`, and `partnerName`.